cmd/archive.go

```diff
@@ -35,6 +36,12 @@ func (c *cmdArchive) run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if c.excludeEnv {
+		c.gs.logger.Debug("environment variables will be excluded from the archive")
+
+		arc.Env = map[string]string{}
```

Comment:
:thinking: I think this should be `nil`, not an empty map, so it is encoded as `null` in the `metadata.json`. That way we'll be able to differentiate (if we ever need to) whether an archive really had no environment variables when created (`"env": {}`) or if they were intentionally excluded (`"env": null`).

Reply:
See no issues with that, I can adjust it